.github/workflows/upload-asset-on-release.yml

```diff
@@ -15,6 +15,17 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Validate version number
+        run: |
+          taggedVersion=${{ env.VERSION }}
+          pluginHeaderVersion=`grep "Version:" wp-plugin-web.php | grep -Eo "[0-9\.]*"`
+          pluginConstantVersion=`grep "_PLUGIN_VERSION" wp-plugin-web.php | grep -Eo "[0-9\.]*"`
```

Comment:
This plugin also has a version set in package.json - checking it too here would be ideal.

Reply:
Ah, we should just drop the version number from the `package.json`... there isn't anything we really need it for is there?

Reply:
It's used in the build, to have a build directory that matches the version - for cache busting mainly.

Reply:
https://github.com/newfold-labs/wp-plugin-web/blob/main/webpack.config.js#L4